### ISSUE TITLE
SQL код разместить в репозитории

### ISSUE THREAD

**Issue by SergeyKashkovsky** [Abanamat21/NetCoreWebAPI#9]
Чтобы все было вместе, что касается проекта, предлагаю создать в корне папку `Database` и разместить там SQL примитивы.

**Comment by Abanamat21** [comment]
Я размышлял о том, чтобы сделать отдельный статический класс с SQL запросами. Сделаю.
Ты пишешь о том, что нужно создать папку, имеется в виду, что в папке будет класс с SELECT запросами, отдельный с INSERT запросами и т. д.?

**Comment by SergeyKashkovsky** [comment]
Да, файлики с SQL кодом положи в папку отдельную в этом репозитории

**Comment by Abanamat21** [comment]
Вынес все SQL операции в отдельный файл.
